modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -99,16 +107,15 @@ sudo curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/lat
 ----
 sudo chmod +x rtfctl
 ----
-. If you are not using the latest version of the cluster-level components (installer package), upgrade to the latest version:
+. If you are not using the latest version of the appliance, upgrade to the latest version:
 
 .. Make sure the `/opt/anypoint/runtimefabric/installer` directory has a minimum of 4 GiB of disk space available.
 .. Verify you have root privileges. Root privileges are required to perform an upgrade.
 .. Verify your environment meets the prerequisites:
 
-*** A production configuration of Runtime Fabric, with a minimum of 3 controller VMs, is required.
-*** The internal load balancer must be running on at least 3 replicas to maintain availability.
-*** An external load balancer must be configured to load balance incoming requests to the controller VMs, with a health check configured for TCP port 443.
-*** The CPU and memory resources for at least one worker node must be available in the cluster.
+*** The internal load balancer must be running on at least 3 replicas to maintain availability. On newer versions of Runtime Fabric, the internal load balancer runs on all controller nodes by default.
+*** An external load balancer must be configured to load balance incoming requests to the controller nodes, with a health check configured for TCP port 443.
+*** The CPU and memory resources for at least one additional worker node must be available in the cluster.
 +
 This enables safe removal of a worker node from the cluster to apply upgrades without impacting application availability.
 *** Applications serving inbound requests must be scaled to a minimum of 2 replicas.
```

Comment:
```suggestion
*** Applications serving inbound requests are scaled to a minimum of two replicas.
```

Reply:
The information in this topic is becoming too deeply nested, which is an indication that the topic should be broken up into two or more peers or contain subtopics.
Not something you can do right now, but you should look at this later.